admin/query/src/main/java/org/codice/ditto/replication/admin/query/status/fields/ReplicationStats.java

```diff
@@ -0,0 +1,204 @@
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ditto.replication.admin.query.status.fields;
+
+import com.google.common.collect.ImmutableList;
+import java.util.Date;
+import java.util.List;
+import javax.annotation.Nullable;
+import org.codice.ddf.admin.api.Field;
+import org.codice.ddf.admin.common.fields.base.BaseObjectField;
+import org.codice.ddf.admin.common.fields.base.scalar.LongField;
+import org.codice.ddf.admin.common.fields.common.PidField;
+import org.codice.ditto.replication.admin.query.replications.fields.Iso8601Field;
+import org.codice.ditto.replication.admin.query.replications.fields.ReplicationStatus;
+
+public class ReplicationStats extends BaseObjectField {
+
+  private static final String FIELD_NAME = "stats";
+
+  private static final String FIELD_TYPE_NAME = "ReplicationStats";
+
+  private static final String DESCRIPTION = "Contains various statistics of a replication's run.";
+
+  private PidField pid;
+
+  private Iso8601Field startTime;
+
+  private Iso8601Field lastRun;
+
+  private Iso8601Field lastSuccess;
+
+  private LongField duration;
+
+  private ReplicationStatus status;
+
+  private LongField pushCount;
+
+  private LongField pullCount;
+
+  private LongField pushFailCount;
+
+  private LongField pullFailCount;
+
+  private LongField pushBytes;
+
+  private LongField pullBytes;
+
+  public ReplicationStats() {
+    super(FIELD_NAME, FIELD_TYPE_NAME, DESCRIPTION);
+    pid = new PidField();
+    startTime = new Iso8601Field("startTime");
+    lastRun = new Iso8601Field("lastRun");
+    lastSuccess = new Iso8601Field("lastSuccess");
+    duration = new LongField("duration");
+    status = new ReplicationStatus();
+    pushCount = new LongField("pushCount");
+    pullCount = new LongField("pullCount");
+    pushFailCount = new LongField("pushFailCount");
+    pullFailCount = new LongField("pullFailCount");
+    pushBytes = new LongField("pushBytes");
+    pullBytes = new LongField("pullBytes");
+
+    pid.isRequired(true);
+    startTime.isRequired(true);
+    lastRun.isRequired(true);
+    duration.isRequired(true);
+    status.isRequired(true);
+    pushCount.isRequired(true);
+    pullCount.isRequired(true);
+    pushFailCount.isRequired(true);
+    pullFailCount.isRequired(true);
+    pushBytes.isRequired(true);
+    pullBytes.isRequired(true);
+  }
+
+  @Override
+  public List<Field> getFields() {
+    return ImmutableList.of(
+        pid,
+        startTime,
+        lastRun,
+        lastSuccess,
+        duration,
+        status,
+        pushCount,
+        pullCount,
+        pushFailCount,
+        pullFailCount,
+        pushBytes,
+        pullBytes);
+  }
+
+  public void setPid(String pid) {
```

Comment:
Don't we normally do `pid(String pid)` and `pid()` for graphql fields

Reply:
It doesn't matter either way, but I personally prefer the java get/set standard. IMO it plays better with code integration tools.